core/src/main/java/com/scalar/db/transaction/consensuscommit/ConsensusCommit.java

```diff
@@ -42,15 +42,13 @@ public class ConsensusCommit extends AbstractDistributedTransaction {
   private final CommitHandler commit;
   private final RecoveryHandler recovery;
   private Runnable beforeRecoveryHook;
-  private Runnable beforeCommitHook;
```

Comment:
Deleted `beforeCommitHook`. 

We used it in the integration test and executed another transaction in this hook. But, actually, instead of using it, we can simply execute another transaction before commit the main transaction. That's why I deleted it.